scripts/whisper/rknn/test_torch.py

```
@@ -0,0 +1,144 @@
+if __name__ == "__main__":
+    torch.set_num_threads(1)
+    torch.set_num_interop_threads(1)
+    try:
+        # To fix
+        # TypeError: scaled_dot_product_attention(): argument 'is_causal' must be bool, not Tensor
+        # See also https://github.com/k2-fsa/sherpa-onnx/issues/1764
+        from whisper.model import disable_sdpa
+
+        with disable_sdpa():
+            main()
+    except:
```

Flake8 (7.3.0)

[error] do not use bare 'except'

(E722)